src/sat.cc

```diff
@@ -2093,25 +2121,6 @@ bool Sat::Cleanup() {
   return true;
 }
 
-// Pretty print really obvious results.
-bool Sat::PrintResults() {
-  bool result = true;
-
-  logprintf(4, "\n");
-  if (statuscount_) {
-    logprintf(4, "Status: FAIL - test encountered procedural errors\n");
-    result = false;
-  } else if (errorcount_) {
```

Comment:
Are these variables used for anything else ? Should we just log these somewhere? Or take these off? Or do nothing ?

Reply:
Status count has been removed. I do log error count as a measurement, perhaps not in this PR